spec/embedded-protocol.md

```diff
@@ -257,29 +257,16 @@ the same equality semantics can be generated for a number `x` by rounding
 
 ### Colors
 
-The protocol includes three distinct color value types, `RgbColor`, `HslColor`,
-and `HwbColor`. In Sass code and custom functions, colors may be represented or
-manipulated in either RGB, HSL, or HWB form, so having multiple types allows
-whichever form is currently in use to be sent between endpoints without having
-to eagerly normalize it.
-
-However, users of the host language API should be able to transparently treat
-any color object as though it were either RGB, HSL, or HWB form. The API should
-provide access to the red, green, and blue, hue, saturation, lightness,
-whiteness, and blackness channels of *every* color object. It should use [this
-RGB-to-HSL algorithm], [this HSL-to-RGB algorithm], [this RGB-to-HWB algorithm],
-and [this HWB-to-RGB algorithm] to convert between representations as necessary.
-
-[this RGB-to-HSL algorithm]: https://en.wikipedia.org/wiki/HSL_and_HSV#RGB_to_HSL_and_HSV
-[this HSL-to-RGB algorithm]: https://www.w3.org/TR/css3-color/#hsl-color
-[this RGB-to-HWB algorithm]: https://www.w3.org/TR/css-color-4/#rgb-to-hwb
-[this HWB-to-RGB algorithm]: https://www.w3.org/TR/css-color-4/#hwb-to-rgb
-
-The API should also provide means of changing one or more channels of a color
-while leaving other channels as-is.
-
-Two colors are equal if their RGB forms have the same red, green, blue channels
-and alpha channels within 1e-11 of one another.
+Colors are represented by the `Color` value type. The API should provide means
+of changing one or more channels of a color while leaving other channels as-is.
+
+Two [legacy colors] are equal if their RGB forms have the same red, green, blue
+channels and alpha channels within 1e-11 of one another.
```

Comment:
"within 1e-11" is actually outdated—we changed that logic since it's not properly transitive. It's now "equal rounded to the nearest 1e-11".

Reply:
Addressed in https://github.com/sass/sass/pull/3718/commits/666b693a1b9f87fb252389516f05b751f0b7ff84